docs/products/kafka/howto/manage-acls.rst

```diff
@@ -0,0 +1,53 @@
+Managing users and access control lists
+=======================================
+
+Aiven for Apache Kafka supports the definition of users and access control lists (ACL) to allow or limit producer and consumer rights at the topic.
```

Comment:
" supports the definition of users and access control lists (ACL)" might be a stable phrase, but if it is not, would it help if we rephrase " supports the user definition and access control lists (ACL)".
Also the whole sentence is a bit difficult to process.
I don't have a perfect option, but what about "Aiven for Apache Kafka uses access control lists (ACL) and user definitions in order to establish individual rights to produce or consume a topic"? :) of course, disregard it, if this skews the meaning.

Reply:
This is used in three places, btw, my comment applies to all of them

Reply:
fixed as suggested